ketcher-autotests/playwright.config.ts

```diff
@@ -17,8 +17,16 @@ const ignoredTests = [
   'API/**',
   'File-Management/Smile-Files/smile-files.spec.ts',
   'Examples/**',
-  'Indigo-Tools/**',
-  'R-group-tool/**',
+  'Indigo-Tools/Clean-Tools/clean-tools.spec.ts:20',
+  'Indigo-Tools/Clean-Tools/clean-tools.spec.ts:36',
+  'Indigo-Tools/Clean-Tools/clean-tools.spec.ts:94',
+  'Indigo-Tools/Clean-Tools/clean-tools.spec.ts:107',
+  'Indigo-Tools/Clean-Tools/clean-tools.spec.ts:124',
+  'Indigo-Tools/Clean-Tools/clean-tools.spec.ts:138',
+  'Indigo-Tools/Clean-Tools/clean-tools.spec.ts:158',
+  'Indigo-Tools/Clean-Tools/clean-tools.spec.ts:215',
+  'Indigo-Tools/Clean-Tools/clean-tools.spec.ts:232',
```

Comment:
how about using test.skip in spec file instead describe it here?
it allows us change tests inside spec file without changing number of string here.

Reply:
Yeah, I added 'fixme'